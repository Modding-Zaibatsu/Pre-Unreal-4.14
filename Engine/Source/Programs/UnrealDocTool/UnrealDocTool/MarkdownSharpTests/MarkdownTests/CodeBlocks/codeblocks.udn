    #include <iostream>

    int main(void)
	{
	    std::cout << "Hello World!" << std::endl;
	}

`Inline code block` paragraph end.

1. List item

		Code block with a tab at the end. The tab is important for the test purpose.	

		
	Last code block in the file
	has to be two-lines code block.