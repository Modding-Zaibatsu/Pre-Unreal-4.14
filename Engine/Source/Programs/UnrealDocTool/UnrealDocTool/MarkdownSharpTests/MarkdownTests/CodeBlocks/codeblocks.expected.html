<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" dir="ltr" lang="en-US">
	<head>
	</head>
	<body>
		<pre class="prettyprint"><code>#include &lt;iostream&gt;

int main(void)
{
    std::cout &lt;&lt; "Hello World!" &lt;&lt; std::endl;
}</code></pre>
        
        <p><code>Inline code block</code> paragraph end.</p>
        
	    <ol >
	        <li>
	            <p>List item</p>
                <pre class="prettyprint"><code>Code block with a tab at the end. The tab is important for the test purpose.</code></pre>
	        </li>
	    </ol>
        
	    <pre class="prettyprint"><code>Last code block in the file
has to be two-lines code block.</code></pre>
	</body>
</html>
